### DISCUSSION TITLE
My nodejs script is terminating every so often

### DISCUSSION THREAD

**Discussion by NoHaxito** [community/community#52568] [Codespaces]
### Select Topic Area

Bug

### Body

Hi, i have an codespaces, this works fine, im happy with codespaces, but when i run an nodejs script (yarn dev or npm run dev for example) this is finished out of nowhere every 2 minutes as if you did Ctrl + C in terminal.
I attach an image of evidence.
![image](https://user-images.githubusercontent.com/83735848/231306358-ac82457d-d7d4-46bb-ac89-59f0756d67eb.png)
![image](https://user-images.githubusercontent.com/83735848/231306392-83600c5c-4adf-4139-a25a-bb5be51392c9.png)

And I could attach many more screenshots.

This problem/bug began to occur a week ago, since it continues to occur, I wanted to know
if it has a solution and how can I fix it 
### EDIT
Now this time the script has dured 40 seconds live hahaah
![image](https://user-images.githubusercontent.com/83735848/231307444-0636f62d-c4f2-4f27-9a03-c3e746b1f8bb.png)

**Comment by joshaber** [comment]
Hi,

I'm not totally clear on what the problem is. Is it that the script is taking too long to run? Or that the script is intended to keep running but is instead quitting? Is it possible the machine is running out of memory and so the script is getting terminated?

**Reply by NoHaxito** [reply]
The memory problem could be believed but the script is exiting instead of running for 1 week and a half.
I'll see if changing the region might fix the problem
